src/models/message/LibraryResponse.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Estonian Information System Authority
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { SignatureAlgorithm } from "../SignatureAlgorithm";
+
+export interface LibraryStatusResponse {
+  /**
+   * A version number string in SemVer format
+   *
+   * @see https://semver.org/
+   */
+  library: string;
+
+  /**
+   * A version number string in SemVer format
+   *
+   * @see https://semver.org/
+   */
+  extension: string;
+
+  /**
+   * A version number string in SemVer format
+   *
+   * @see https://semver.org/
+   */
+  nativeApp: string;
+}
+
+export interface LibraryAuthenticateResponse {
+  /**
+   * The base64-encoded DER-encoded authentication certificate of the eID user
+   *
+   * The public key contained in this certificate should be used to verify the signature.
+   * The certificate cannot be trusted as it is received from client side and the client can submit a malicious certificate.
+   * To establish trust, it must be verified that the certificate is signed by a trusted certificate authority.
+   */
+  unverifiedCertificate: string;
+
+  /**
+   * The algorithm used to produce the authentication signature
+   *
+   * The allowed values are the algorithms specified in JWA RFC8 sections 3.3, 3.4 and 3.5
```

Comment:
Ditto.

Reply:
Will change it with the upcoming documentation PR